amethyst_core/src/lib.rs

```diff
@@ -2,20 +2,25 @@
 extern crate cgmath;
 extern crate fnv;
 extern crate hibitset;
+extern crate rayon;
 #[macro_use]
 extern crate serde;
 extern crate shred;
 extern crate specs;
 
-#[cfg(test)]
-#[cfg_attr(test, macro_use)]
-extern crate quickcheck;
+//#[cfg(test)]
```

Comment:
Why is this commented out ?

Reply:
There's not a single usage in the whole crate, I'm not sure what @Aceeri did here.

Reply:
Ah, seems he removed all the tests using it. Remove it altogether then I'd say.